modules/installation-gcp-permissions.adoc

```diff
@@ -45,7 +45,7 @@ the following role:
 * Service Account Key Admin
 
 The roles are applied to the service accounts that the control plane and compute
-machines use:
+machines use. The roles are required for configuring a GCP project to host the {product-title}.
```

Comment:
The initial ask was to clarify "if the entries in the table is for Installer or after-install operations or both". I don't think the above statement answers the question, could you please clarify? Thanks!

Reply:
@jianli-wei Do you have any suggestions for the text? My understanding is that these roles are required to configure a GCP project which is done prior to installing OCP.

Reply:
@sagidlow Sorry, it's not for configuring a GCP project, because the GCP projects (one as **service** project and another as **host** project) should have been created before OCP installation. As for the initial ask, i.e. "_if the entries in the table is for Installer or after-install operations or both_", sorry that I cannot answer with sure even after discussing with other QEs. We guess, the Day-1 and Day-2 operations may require the same set of roles, in terms of the control plane and compute machines. @patrickdillon Would you please advise? Thanks!

Reply:
Personally, I don't think the bug is valid. It says "As it is written now, it is not clear if the entries in the table is for Installer or after-install operations or both." The doc clearly states: " To deploy an {product-title} cluster, the service
account requires the following permissions."

So yes, it is a requirement for deployment. We have separate sections which also talk about permissions required to operate the cluster. I think https://bugzilla.redhat.com/show_bug.cgi?id=1961399 is NOTABUG

Asa a side note, when reviewing this, I noticed "The roles are applied to the service accounts that the control plane and compute machines use." This is not accurate. It could simply be removed.

Reply:
@jianli-wei, are you ok with us closing this issue as NOTABUG?